browser/IgBlazorSamples.Test/Infrastructure/BlazorTest.cs

```diff
@@ -0,0 +1,75 @@
+﻿using Microsoft.AspNetCore.Mvc.Testing;
+using Microsoft.Playwright.NUnit;
+
+namespace IgBlazorSamples.Test;
+
+public class BlazorTest : PageTest
+{
+    protected static readonly Uri RootUri = new("http://127.0.0.1");
+    private readonly WebApplicationFactory<Program> _webAppFactory = new();
+    private HttpClient? _httpClient;
+
+    [SetUp]
+    public async Task BlazorSetup()
+    {
+        _httpClient = _webAppFactory.CreateClient(new()
+        {
+            BaseAddress = RootUri,
+        });
+
+        await Context.RouteAsync($"{RootUri.AbsoluteUri}**", async route =>
```

Comment:
Would appreciate some comments about this basically re-routing all Playwright requests through the in-memory app created by WebApplicationFactory